www/md_base/guanlecoja/config.coffee

```diff
@@ -88,4 +88,33 @@ gulp.task 'icons', ->
         ))
         .pipe(gulp.dest(path.join(config.dir.build, 'icons')))
 
+gulp.task 'apiproxy', ->
+    # this is a task for developing, it proxy api request to http://nine.buildbot.net
+    argv = require('minimist')(process.argv)
+    argv.apiserver ?= 'nine.buildbot.net'
```

Comment:
are you sure you want to hardcode this url?
one might also want to use its own server on localhost